shared/constants/types/whats-new.tsx

```diff
@@ -0,0 +1,14 @@
+export type NoVersion = '0.0.0'
+export type CurrentVersion = '2.3.3'
+export type LastVersion = NoVersion
+export type LastLastVersion = NoVersion
+export type WhatsNewVersion = CurrentVersion | LastVersion | LastLastVersion | NoVersion
+export type WhatsNewVersions = [CurrentVersion, LastVersion, LastLastVersion]
```

Comment:
I decided to turn versions into Type literals as well just to throw a compiler error if there was mismatch. Might be overdoing it but since bumping version numbers is a manual process right now, wouldn't want to accidentally sync the wrong version string via gregor